src/admin-views/metabox/compatibility/events-control-extension/mark-online.php

```diff
@@ -22,7 +22,7 @@
  */
 
 ?>
-<div class="tribe-events-control-metabox-container" style="margin-top: 24px;">
+<div class="tribe-events-control-metabox-container" style="margin-top: 18px;">
```

Comment:
Does this have to be an inline style?

Reply:
It is compatibility coding and will be removed one day so I'd rather not add an entry in the css.